### ISSUE TITLE
sleeping.png transparent wallpaper support after 20-minute timeout

### ISSUE THREAD

**Issue by technicalguy** [ddvk/remarkable-hacks#301]
I commented this on another issue (#157), but it is already closed so I am opening a new issue.

>My sleeping.png has working transparency only if I press the sleep button. If I leave it for 20min to auto-sleep then it appears against a white background. :cry: 
>
>P.S. Everything else about remarkable-hacks is absolutely amazing and I couldn't live without it! That's why I'm a sponsor!

**Comment by dstango** [comment]
are you sure sleeping.png and suspended.png both have transparency? There are different files used for the two situations.

It's working fine for me...

**Comment by technicalguy** [comment]
Ahh, I only had a suspended.png and not a sleeping.png. I've made a sleeping.png now and I will report back.

**Comment by technicalguy** [comment]
Hm, I thought it worked, but just today it didn't seem to work :/

**Comment by StLa1** [comment]
Works for me using [those](https://drive.google.com/drive/folders/1GViubYNpVLrnSt-aJ_QrsBZ1X5ctdklH?usp=sharing)

**Comment by technicalguy** [comment]
> Works for me using [those](https://drive.google.com/drive/folders/1GViubYNpVLrnSt-aJ_QrsBZ1X5ctdklH?usp=sharing)
– @StLa1 

So if you wait for 20min you see the mostly transparent image that says “Sleeping Mode” (`sleeping.png`)? And the transparency works?

My sleeping.png works fine when I manually press the power button, but not if I wait for 20min.

**Comment by StLa1** [comment]
Yes, there will be a black bar at the bottom, but the rest of the screen shows the content that was displayed before sleep

**Comment by technicalguy** [comment]
I think it may have been an issue with trying to use remux at the same time

**Comment by dominique-unruh** [comment]
I can confirm @technicalguy's issue. If the remarkable goes to sleep via timeout, it shows only a black bar on top (saying press a button). I have both sleeping.png (created it myself, wasn't there before) and suspend.png. Pressing the power button shows suspend.png. sleeping.png is never shown.

I run 2.10.3.379-patch_28.2.01-0-ge63b24f

**Comment by FlorianKempenich** [comment]
I wasn't sure where the best place was to ask this, but...  
...I can't get either auto-sleep or button-sleep to work with transparent `sleeping.png` or `suspended.png`. 

Is there anything extra to do? (besides installing `remarkable-hacks`)

**Comment by FlorianKempenich** [comment]
**Update:** I figured it out, both the 20-min screens and the click-to-lock screens are working for me. Here were the problems
- _click-to-lock screen / `suspended.png`:_ This one only works if you have **no passcode** set up, if there's a passcode, the transparent section will appear blank.
- _20-min screen / `sleeping.png`:_ This one seems to be cached, make sure to **reboot** after copying. (also maybe needs no passcode, not 100% sure)

**Comment by FlorianKempenich** [comment]
Maybe we could add the info to the `README.md` and close this issue? 

I don't mind doing a PR but I'm not sure where would be the best place to put that info. What do you think @ddvk ?

**Comment by martin-braun** [comment]
@FlorianKempenich Like so? #422

**Comment by FlorianKempenich** [comment]
@martin-braun Yep, that looks good. I think I had searched the term "transparent" on the readme when I had this problem. So, with your update, I would have found the info 👍

**Comment by martin-braun** [comment]
@FlorianKempenich "transparent" would not hit though, I rather use "transparent support" instead of "transparency support" then.

**Comment by FlorianKempenich** [comment]
@martin-braun That's a good point, but on most browser it would still kind of work: As we start typing the word "transparen..." will get highlighted, that should be enough. 

That being said, now that I think about it, my first thought when I looked for this feature was _not_ actually to use the term 'transparent' or 'transparency', I went for these terms after a bit of research. Originally I was looking for something like: "Remove lock screen", or "View notes when tablet is locked". So, something around that might be even more useful to the end-user. Because, in the end, that's the real feature/goal. What do you think? 🙂

**Comment by martin-braun** [comment]
@FlorianKempenich Now you are going for the SEO route, don't you? ;) This hack has plenty of features to offer without having good SEO about it, especially when it comes to the gestures. People who look for transparent support would know about those file names and perform a better Google Search to match our phrase, don't you think?

"reMarkable View notes when tablet is locked" is not a good search for this feature, because it will result in hits for things like the lock screen and this feature is not dedicated for viewing notes while the screen is locked, but also to have cool art that allows to look through: 

![2f1jsp5ovig61](https://user-images.githubusercontent.com/24259245/156031816-c2ba18c2-5a3c-4a0b-89af-491d903f6bda.jpeg)

And ddvk's hack [is mentioned on there](https://www.reddit.com/r/RemarkableTablet/comments/lgecv5/my_custom_transparent_sleep/gmsg5lf/?utm_source=reddit&utm_medium=web2x&context=3) as well.

I would say it's good to go, short and precise. You are still free to make a PR to improve further. :)

**Comment by FlorianKempenich** [comment]
Didn't really think about SEO no. Was just trying to think what would have been helpful when I looked for it. 

But sure, the current version is better than nothing at all. Thanks for the contribution 👍
